### DISCUSSION TITLE
Have social share image, but do not display in blog post

### DISCUSSION THREAD

**Discussion by bee-san** [adityatelange/hugo-PaperMod#155] [Q&A] [answered]
Hey,

When you add a social share image, it displays in the blog post:
![image](https://user-images.githubusercontent.com/10378052/103152420-acc5e080-477f-11eb-84c1-941e66b62a25.png)

I want the image to display in social share cards as [Summary with large image](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image), but I don't want it to display in the blog posts themselves (as Lighthouse complains about speed, and I think it's ugly).

How do I disable this? :)

**Comment by adityatelange** [comment, marked as answer]
Sure, we'll add a var to hide it, both for list as well as on single, no ETA tho :)
